utils/docker/Dockerfile.centos.7

```diff
@@ -73,8 +73,9 @@ RUN yum -y install \
            python2-clustershell python2-Cython                 \
            python36-clustershell python36-paramiko             \
            python36-numpy python36-jira python3-pip Lmod       \
-           fuse3-devel libipmctl-devel openmpi3-devel          \
+           fuse3-devel libipmctl-devel                         \
            hwloc-devel patchelf python36-tabulate java-1.8.0-openjdk
+RUN yum -y install openmpi3-devel
```

Comment:
How does installing this separately alleviate the conflict between openmpi and MOFED?  And how is MOFED being installed given what's in this `Dockerfile`?

Reply:
The command will fail and the docker install will just go on, no?

Reply:
1. Installing openmpi3-devel separately doesn't alleviate the conflict with MOFED.  It alleviates the problem that all other RPMs on that "yum install" will **also** not be installed. Separating them allows all the other RPMs to install cleanly, and **just** the openmpi3-devel install fails. 

2. I am not installing within Docker, this is for a barebone CentOS install, where MOFED installation is happening befor installing DAOS. The install guide advises to use the "yum install" steps from the Dockerfile (without the "RUN " prefix), that's why the conflict with MOFED is showing here...